docusaurus/docs/importing-a-component.md

```diff
@@ -48,3 +48,28 @@ Learn more about ES6 modules:
 - [When to use the curly braces?](https://stackoverflow.com/questions/36795819/react-native-es-6-when-should-i-use-curly-braces-for-import/36796281#36796281)
 - [Exploring ES6: Modules](http://exploringjs.com/es6/ch_modules.html)
 - [Understanding ES6: Modules](https://leanpub.com/understandinges6/read#leanpub-auto-encapsulating-code-with-modules)
+
+## Absolute Imports
+
+You can configure your application to support importing modules using absolute paths. This can be done by configuring a `jsconfig.json` or `tsconfig.json` file in the root of your project. If you're using TypeScript in your project, you will already have a `tsconfig.json` file.
+
+Below is an example `jsconfig.json` file for a JavaScript project. You can create the file if it doesn't already exist:
+
+```json
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  },
+  "include": ["src"]
```

Comment:
Do you need to specify `src` here? I haven't tested, just wondering... if not, we could simplify this.

Reply:
Not for absolute imports. It’s used to specify which files should be included for TypeScript compilation and/or which files should be analyzed by the IDE.